### DISCUSSION TITLE
Can I join?

### DISCUSSION THREAD

**Discussion by zoraezimran** [theforumhelpers/theforumhelpers.github.io#139] [Q&A] [answered]
I would like to join your organization I am a scratcher too.

pls can I join

**Comment by Accio1** [comment, marked as answer]
As of right now, only Scratchers who are a part of [The Forum Helpers](https://scratch.mit.edu/studios/3688309/) can join this organisation. However, you are still able to open issues and create pull requests even if you aren't a Forum Helper.
